features/step-definitions/sqlup-mode.el-steps.el

```diff
@@ -39,3 +39,10 @@
   (lambda (mode)
     (let ((v (vconcat [?\C-u ?\M-x] (string-to-vector mode) [?\C-m])))
 (execute-kbd-macro v))))
+
+(When "^I mock turn on sql-interactive-mode$"
```

Comment:
This is cool. You should consider opening a PR on https://github.com/ecukes/espuds with that :)